build.sbt

```diff
@@ -139,6 +206,16 @@ lazy val app = mainProject("app")
       case "module-info.class" =>
         MergeStrategy.discard
       case other => (assembly / assemblyMergeStrategy).value(other)
+    },
+    apiMappings ++= {
+      val slickAPIMapping =
```

Comment:
we could maybe have this `val slickAPIMapping` define outside so we can reuse it here and below

Reply:
Yep that would've been much cleaner. But sbt doesn't allow it. 

> error: `value` can only be called on a task within a task definition macro
classPath    = (Compile / fullClasspath).value

I'm sure there is a way to make this work, it's Scala after all. Will try to figure it out another PR.